live-build/build.gradle

```diff
@@ -0,0 +1,112 @@
+/*
+ * Copyright 2019 Delphix
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+apply plugin: "base"
+
+apply from: "${rootProject.projectDir}/gradle-lib/util.gradle"
+
+task ancillaryRepository(type: Exec) {
+    inputs.file "${rootProject.projectDir}/scripts/build-ancillary-repository.sh"
+
+    for (envVar in ["AWS_S3_URI_VIRTUALIZATION",
```

Comment:
does this mean, we now have to updated two places when exposing environment variables to our scripts? i.e. the `docker-run.sh` file, and also the gradle file(s)?